apps/src/templates/InputPrompt.jsx

```diff
@@ -32,9 +29,15 @@ const styles = {
 export default class InputPrompt extends Component {
   static propTypes = {
     question: PropTypes.string.isRequired,
-    onInputReceived: PropTypes.func.isRequired
+    onInputReceived: PropTypes.func.isRequired,
+    currentValue: PropTypes.string
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {value: this.props.currentValue};
+  }
```

Comment:
[same comment as above about setting initial state at the component level to remove the constructor]